constants.py

```diff
@@ -0,0 +1,48 @@
+
+K_COLOUR_V_FOOD_DICT = {
+  "purple": "grapes",
+  "red": "apple",
+  "brown": "bread",
+  "yellow": "cheese",
+  "green": "pepper"}
+"""Cross reference which food token matches with which ant colour
+
+A dictionary to refer to when cross referencing is required between 
+ant colour and food type.
+Keys are ant colours as strings
+Values are food IDs as strings
+"""
+
+
+ANTS = K_COLOUR_V_FOOD_DICT.keys()
+# ANTS = ['purple', 'red', 'brown', 'yellow', 'green']
+"""A list of ID for the insect meeple
```

Comment:
Note the nuance that ANTS is a group of keys, rather than a list. Which is similar but slightly different. to convert to list, you can do `ANTS = list(K_COLOUR_V_FOOD_DICT.keys())`